src/Utilities/TagsModal.js

```diff
@@ -82,9 +82,11 @@ const TagsModal = ({
     setFilterBy(filterTagsBy);
   }, [filterTagsBy]);
 
+  /* eslint-disable react-hooks/exhaustive-deps */
   useEffect(() => {
     setSelected(preselectedTags);
   }, [JSON.stringify(preselectedTags)]);
```

Comment:
NIT: here I think it wouldn't hurt to add `setSelected` too, but I don't mind this solution either.